std/format/package.d

```diff
@@ -269,119 +269,242 @@ It's also possible to use positional arguments for $(I width), $(I
 precision) and $(I separator) by adding a number and a $(B
 '$(DOLLAR)') after the $(B '*').
 
+$(SECTION4 Types)
+
+The following table contains detailed descriptions for every
+available combination of type and format character. To avoid
+unnecessary repetition, here are some notes that apply to all or
+several entries:
```

Comment:
I would reorganize this paragraph the following way:

- start by explaining what this section does. E.g: "This section describes the result of combining types with format characters. It is organized in 2 subsections: a list of general information regarding the formatting of types in the presence of format characters and a table that contains details for every available combination of type and format character".
- continue with the general description paragraph: "When formatting types, the following rules apply:".
- finish with the table: "This table contains descriptions for every possible combination of type and format character"